### ISSUE TITLE
Update tutorial to avoid use of copy.deepcopy()- Convolution/Batch Norm Fuser in FX

### ISSUE THREAD

**Issue by sekyondaMeta** [pytorch/tutorials#2331]
Update the  [Building a convolution/batch norm fuser in FX](https://github.com/pytorch/tutorials/blob/6ae56a1a2639308550a69c5006e879d331f05b6d/intermediate_source/fx_conv_bn_fuser.py#L107) to avoid the use of copy.deepcopy().
Tutorial link: https://github.com/pytorch/tutorials/blob/6ae56a1a2639308550a69c5006e879d331f05b6d/intermediate_source/fx_conv_bn_fuser.py#L107

See issue #2177  for more information

cc @eellison @suo @gmagogsfm @jamesr66a @msaroufim @SherlockNoMad @albanD @svekars @carljparker @NicolasHug @kit1980 @subramen

**Comment by svekars** [comment]
This issue has been unassigned due to inactivity. If you are still planning to work on this, you can still send a PR referencing this issue.
